pyo3-build-config/src/impl_.rs

```diff
@@ -20,7 +20,7 @@ use crate::{
 /// Minimum Python version PyO3 supports.
 const MINIMUM_SUPPORTED_VERSION: PythonVersion = PythonVersion { major: 3, minor: 7 };
 /// Maximum Python version that can be used as minimum required Python version with abi3.
-const ABI3_MAX_MINOR: u8 = 9;
+const ABI3_MAX_MINOR: u8 = 10;
```

Comment:
Isn't this a bug in the released implementation? Should this become a separate PR with a change log entry or is it inconsequential?

Reply:
Yes, I think we should split this out & aim to include in next patch release.

Reply:
Right, but if they try to compile for Python 3.10 with the `abi3-py310` feature then I think we'll accidentally be using the 3.9 abi, thus missing out on newer features.